config/locales/en.yml

```diff
@@ -294,7 +294,9 @@ en:
     otp_code: OTP code
     require_mfa_disabled: Your multi-factor authentication has been enabled. To reconfigure multi-factor authentication, you'll have to disable it first.
     require_mfa_enabled: Your multi-factor authentication has not been enabled. You have to enable it first.
+    setup_required: For protection of your account and your gems, you are required to set up multi-factor authentication.
```

Comment:
Users might be confused on why MFA is being required, could we link the blog post here?(https://blog.rubygems.org/2022/08/15/requiring-mfa-on-popular-gems.html)